samples/solidity/contracts/ERC721WithData.sol

```diff
@@ -23,7 +24,17 @@ import './IERC721WithData.sol';
  * This is a sample only and NOT a reference implementation.
```

Comment:
Comments above need to be updated to describe the new URI functionality (not hardcoded)